sorc/ncl.setup

```diff
@@ -1,28 +1,12 @@
-if [ $target == wcoss_dell_p3 ]
-then
-module load NCL/6.4.0 
-fi
-
-if [ $target == wcoss_cray ]
-then
-module load   gcc/4.9.2
-module load NCL-gnu-haswell/6.3.0
-fi
-
-if [ $target == wcoss ]
-then
-module load ncarg/v6.1.0
-fi
-
-if [ $target ==  jet ]
-then
-module load  ncl/6.5.0 
-export NCARG_LIB=$NCARG_ROOT/lib
-fi
-
-if [ $target ==  hera ]
-then
-module load  ncl/6.5.0 
-export NCARG_LIB=$NCARG_ROOT/lib
-fi
-
+#!/bin/bash
```

Comment:
I don't think this script is even used anymore. Loading the NCL module should take care of it.

Bigger picture, we need to have people port their NCL codes to python since NCL has been deprecated (I just opened issue #923)

Reply:
At this time, this PR is removing deprecated WCOSS related references.  
We can get broader engagement and then remove this file then.